### DISCUSSION TITLE
Issues Installing Foundry - libusb not found

### DISCUSSION THREAD

**Discussion by beeski00** [Cyfrin/foundry-full-course-cu#791] [Q&A] [answered]
Hey All,

Trying to install foundry. I am getting the following error  after giving it the foundry code: curl -L https://foundry.paradigm.xyz | bash

Error: libusb not found. You may need to install it manually on MacOS via Homebrew (brew install libusb).

I then tried to do the install command of $ brew install libusb ... no luck.

I then went to here https://github.com/libusb/libusb/releases. And downloaded [libusb-1.0.26-binaries.7z](https://github.com/libusb/libusb/releases/download/v1.0.26/libusb-1.0.26-binaries.7z). And then tried to do the install command. Still no luck.

Has anyone seen this issue? or know next steps to troubleshooting.

Please let me know.

Beeski

**Comment by theirrationalone** [comment, marked as answer]
Heyy @beeski00 ! Please Follow this [discussion](https://github.com/Cyfrin/foundry-full-course-f23/discussions/729)

**Reply by beeski00** [reply]
Thank you Anil!!!!!
